CONTRIBUTING.md

```diff
@@ -1,9 +1,10 @@
 # Contributing to Node.js
 
-* [Code of Conduct](#code-of-conduct)
-* [Issues](#issues)
-* [Pull Requests](#pull-requests)
-* [Developer's Certificate of Origin 1.1](#developers-certificate-of-origin)
+- [Contributing to Node.js](#contributing-to-nodejs)
```

Comment:
This should not be included in the table of contents.

It seems like changes to this file may have been generated automatically. They seem unrelated to the rest of this pull request.

Reply:
Oh, wait, they are related because it adds a table of contents that wasn't there before. Never mind the previous comment other than the part about not including the level 1 heading in the table of contents (since it is literally the only thing that appears in the file before the table fo contents, so it's effectively listing the table of contents inside the table of contents, which might make sense in a book in some cases, but doesn't make sense in a web page).